src/main/java/org/folio/dataimport/util/RestUtil.java

```diff
@@ -181,4 +181,35 @@ private static boolean isJson(AsyncResult<WrappedResponse> asyncResult) {
     HttpClientResponse response = asyncResult.result().response;
     return response != null && APPLICATION_JSON.equals(response.getHeader(CONTENT_TYPE));
   }
+
+  /**
+   * Checks whether the answer is partial successful
+   *
+   * @param response - http response
+   * @return - true if response status is "server error" and content type is "application/json", otherwise false
```

Comment:
just a note - we better not forget to get rid of this method once partial success status is changed, maybe wee should create a ticket for that?

Reply:
reference to created task for this issue [MODDATAIMP-150](https://issues.folio.org/browse/MODDATAIMP-150)